src/sass/_theme_rst.sass

```diff
@@ -447,6 +447,8 @@
     .property
       display: inline-block
       padding-right: 8px
+      &:after
+        content: " "
```

Comment:
> use `user-select: none` to avoid selecting this element at all.

I think it would be weird to not be able to select it at all (and annoying if you actually did want to copy the whole declaration).



> I actually already have a space after this element when building with a more recent Sphinx release.

It looks like we were using Sphinx 3.5.3. I guess we will have to try 4.x.